======================================== 
Start time: 2025/04/28 周一 22:46:14.06 
Using conda environment: pytorch_gpu 
Starting Python script execution... 
Traceback (most recent call last):
  File "C:\Users\ZY\Desktop\Forecaster\scripts\task_evaluate.py", line 24, in <module>
    Evaluate_Task.execute_eva() # 依据task_id自动选择类方法创建数据文件
  File "C:\Users\ZY\Desktop\Forecaster\scripts\..\src\model_evaluator\evaluate_src.py", line 76, in execute_eva
    return task_mapping[task_id]()
  File "C:\Users\ZY\Desktop\Forecaster\scripts\..\src\model_evaluator\evaluate_src.py", line 44, in eva_task1
    model_class_dic = joblib.load('../data/user_data/{}/test_job_data/model_class_{}_{}_{}.joblib'.format(self.user_id, self.user_id, self.task_id, self.job_number))
  File "C:\ProgramData\Anaconda3\envs\pytorch_gpu\lib\site-packages\joblib\numpy_pickle.py", line 650, in load
    with open(filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '../data/user_data/xiaowang/test_job_data/model_class_xiaowang_1_0.joblib'
 
End time: 2025/04/28 周一 22:46:18.30 
Script execution completed 
